dashboard/client/src/pages/dashboard/node-info/NodeInfo.tsx

```diff
@@ -68,24 +67,17 @@ const makeGroupedTableContents = (
   nodes: Node[],
   sortWorkerComparator: any,
   sortGroupComparator: any,
-  rayletInfo: RayletInfoResponse | null,
   nodeInfoFeatures: NodeInfoFeature[],
 ) => {
   const sortedGroups = sortGroupComparator
     ? stableSort(nodes, sortGroupComparator)
     : nodes;
   return sortedGroups.map((node) => {
-    const plasmaStats = rayletInfo?.plasmaStats?.[node.ip];
     const workerFeatureData: WorkerFeatureData[] = node.workers.map(
       (worker) => {
-        const rayletWorker =
```

Comment:
This is no longer needed because the backend now injects the data into the payload per-actor.